pkg/cmd/miscellaneous.go

```diff
@@ -396,6 +374,11 @@ func getTargetMapInfo() {
 	if isTargeted("project", "shoot") {
 		targetInfo["seed"] = *shoot.Spec.SeedName
 	}
+
+	// user for case garden-seed-shoot add [project] = garden
+	if isTargeted("seed", "shoot") {
+		targetInfo["project"] = "garden"
+	}
```

Comment:
This is not correct. Why do you think you can assume the project name `garden`?

Reply:
fixed